### ISSUE TITLE
retain bind mounts in container

### ISSUE THREAD

**Issue by Xynonners** [Moe-hacker/moe-container#9]
hi,

I'm trying to bind mount termux tmp to container tmp to share X11 socket.
```
su -c 'mount --bind /data/data/com.termux.x11/files/tmp /data/chimera/tmp'
```

after updating to the latest version, bind mounts seem to get overriden and this no longer works.

additionally, using the -m flag doesn't seem to work.
```
-m /data/user/0/com.termux.x11/files/tmp /tmp
```

**Comment by Moe-hacker** [comment]
> hi,
> 
> I'm trying to bind mount termux tmp to container tmp to share X11 socket.
> ```
> su -c 'mount --bind /data/data/com.termux.x11/files/tmp /data/chimera/tmp'
> ```
> 
> after updating to the latest version, bind mounts seem to get overriden and this no longer works.
> 
> additionally, using the -m flag doesn't seem to work.
> ```
> -m /data/user/0/com.termux.x11/files/tmp /tmp
> ```

Thanks for reporting, this might be fixed at 95791de2f4e9d4937f2310f358e58ee1f373439e
By the way, since the new version is working in process, it's recommended to back to v2.7

**Comment by Xynonners** [comment]
yup, fixed.
